akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1770,6 +1824,7 @@ object NoMarkerLogging extends MarkerLoggingAdapter(null, "source", classOf[Stri
   final protected override def notifyError(message: String): Unit = ()
   final protected override def notifyError(cause: Throwable, message: String): Unit = ()
   final protected override def notifyWarning(message: String): Unit = ()
+  final protected override def notifyWarning(cause: Throwable, message: String): Unit = ()
```

Comment:
this will break if a library extended LoggingAdapter and an user upgrades akka.

Reply:
Aw, forgot about the trait forwarders problem, yes :(

Reply:
“This is why we can’t have nice things”